### PR TITLE
ECIP-1075 points to the various Aztlán editions

### PR DESCRIPTION
**Pull request by bobsummerwill**

### CODE REVIEW

**Comment by bobsummerwill** [comment]
@soc1c Your proposal is now ECIP-1075.

Alternatively, I could make ECIP-1075 be "the index".   Let me make a PR for that, so we can see what the diffs look like that way round.

File: _specs/ecip-1072.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 lang: en
 ecip: 1072
-title: Yingchun EVM and Protocol Upgrades
+title: Aztlán EVM and Protocol Upgrades (Yuanchun Edition)
```
Comment (YazzyYaz) [inline _specs/ecip-1072.md]: Typo. Must say (Yingchun Edition)

**Comment by bobsummerwill** [comment]
Obsoleted by https://github.com/ethereumclassic/ECIPs/pull/198.

**Comment by sorpaas** [comment]
I don't understand why this is closed. We certainly need either https://github.com/ethereumclassic/ECIPs/pull/185 or this. Given we decided not to move #185 forward, that means we need to find a way to merge this?

**Comment by bobsummerwill** [comment]
No need.   https://github.com/ethereumclassic/ECIPs/pull/199/files covers it, @sorpaas.

**Pull request closed without merging**